docs/editor/command-line.md

```diff
@@ -102,8 +102,8 @@ There are several CLI options that help with reproducing errors and advanced set
 
 Argument|Description
 ------------------|-----------
-`--extensions-dir <dir>` | Set the root path for extensions. Has no effect in [Portable Mode](/docs/editor/portable.md).
-`--user-data-dir <dir>` | Specifies the directory that user data is kept in, useful when running as root. Has no effect in [Portable Mode](/docs/editor/portable.md).
+`--extensions-dir <dir>` | Set the root path for extensions. (Overridden in [Portable Mode](/docs/editor/portable.md) by the presence of the data folder\*).
```

Comment:
Interesting details in the description of the [original commit](https://github.com/microsoft/vscode-docs/commit/fe4fbc1398a12b5698f836e84d7a1fec2abb56a9). 

> Portable mode (presence of  `/data/` folder in VSC installation directory) effectively prevents `--user-data-dir` and `--extensions-dir` switches from working.

```suggestion
`--extensions-dir <dir>` | Set the root path for extensions. (Overridden in [Portable Mode](/docs/editor/portable.md) by the presence of the data folder in the VS Code installation).
```